include/albatross/src/models/sparse_gp.hpp

```diff
@@ -400,8 +400,65 @@ class SparseGaussianProcessRegression
     return fit;
   }
 
+  template <typename FeatureType>
+  auto fit_from_prediction(const std::vector<FeatureType> &new_inducing_points,
```

Comment:
Is there a better variable name than `new_inducing_points` which suggests the application. What do we use for other `fit_from_prediction` methods? `features_`?

Reply:
I actually did change it to features first ... then realized that whatever gets passed in there become the new inducing points ... so switched back to `new_inducing_points` which I think acts as a clarification since it may not be obvious to someone calling `fit_from_prediction` that the inducing points won't come from the inducing point strategy.